src/test/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogTraceAbstractTest.java

```diff
@@ -49,4 +51,17 @@ protected void assertCleanupActions(Run<?,?> run) {
         assertNull(run.getAction(IsPipelineAction.class));
         assertNull(run.getAction(StepTraceDataAction.class));
     }
+
+    protected void checkHostNameTag(Map meta) {
+        assertNotNull(meta);
+        Object nodeName = meta.get(CITags.NODE_NAME);
+        assertNotNull(nodeName);
+        // if nodeName == master, should be null, otherwise should be none
+        Object hostTagVal = meta.get(CITags._DD_HOSTNAME);
+        if ("master".equals(nodeName)) {
```

Comment:
```suggestion
        if ("master".equals(nodeName) || "built-in".equals(nodeName)) {
```

Aren't we missing the new node name `built-in`?

Reply:
No, this just reflects the logic 
https://github.com/jenkinsci/datadog-plugin/blob/5a44d01b5bd656580463f4eca1dfd54d8609e0a2/src/main/java/org/datadog/jenkins/plugins/datadog/traces/DatadogTraceBuildLogic.java#L148-L152

If we add `built-in`, it will invalidate the check

Reply:
If the master node now has been renamed to `built-in` node, that logic should be modified too

Reply:
This is not needed.